components/mdx-components.tsx

```diff
@@ -14,10 +14,10 @@ import WhatsNew from './WhatsNew'
 import ConnectCommunity from './Connect'
 import { Pagination } from './pagination'
 import LzAlert from './UI/alert'
-import LinkedCard from './GetstartedCard/Cards'
+import LinkedCard, { PlainCard } from './GetstartedCard/Cards'
 import TransferCard from './TransferCard'
 import BankPayouts from './BankPayouts'
-import { props } from 'ramda'
+import { ShopifyIcon, WooCommerceIcon, WordPressIcon } from 'public/icons'
```

Comment:
Let's not import these directly. Instead, create a component that has the cards and its detail and import the component here instead.